pool/web/handler/admin_settings_schedule.ml

```diff
@@ -4,16 +4,21 @@ module Message = HttpUtils.Message
 let src = Logs.Src.create "handler.admin.settings_schedule"
 let active_navigation = "/admin/settings/schedules"
 
-let show =
+let show req =
   Http_utils.Htmx.handler
     ~active_navigation
     ~error_path:active_navigation
     ~query:(module Schedule)
     ~create_layout:General.create_tenant_layout
-  @@ fun context query ->
-  let%lwt schedules, query = Schedule.find_by query in
-  let page = Page.Admin.Settings.Schedule.index context schedules query in
-  Lwt.return (Ok page)
+    (fun context query ->
+      let%lwt schedules, query = Schedule.find_by query in
+      let open Page.Admin.Settings.Schedule in
+      (if HttpUtils.Htmx.is_hx_request req then list else index)
+        context
+        schedules
+        query
+      |> Lwt_result.return)
+    req
```

Comment:
If we prefer this style can I suggest we just pass in the `req` to the `hanlder` function before the function? It'd still allow us to use the `@@ fun ->` pattern which removes some indentation and a set of parentheses?

```ocaml
 Http_utils.Htmx.handler
    req
    ~active_navigation
    ~error_path:active_navigation
    ~query:(module Schedule)
    ~create_layout:General.create_tenant_layout
  @@ fun context query ->
  let%lwt schedules, query = Schedule.find_by query in
  let page = Page.Admin.Settings.Schedule.index context schedules query in
  Lwt.return (Ok page)
```